lib/deployment-utils.ts

```diff
@@ -26,7 +26,7 @@ export const RENVM_PROVIDERS = {
 export const getProvider = (transferRequest) => {
 	if (Object.keys(CONTROLLER_DEPLOYMENTS).includes(transferRequest.contractAddress)) {
 		const chain_key = CONTROLLER_DEPLOYMENTS[transferRequest.contractAddress];
-		return RENVM_PROVIDERS[chain_key](new ethers.providers.JsonRpcProvider(RPC_ENDPOINTS[chain_key]), 'mainnet');
+		return RENVM_PROVIDERS[chain_key](new ethers.providers.JsonRpcProvider(RPC_ENDPOINTS[chain_key]), 'any');
```

Comment:
This should allow for the provider to be on the correct chain based on the RPC provider